### ISSUE TITLE
Take out origin stop and only have destination

### ISSUE THREAD

**Issue by jlawrenceCambridge** [LiveBus/displayV1#4]
For example, if you are headed towards Harvard from MIT, only have Harvard on the line so as not to confuse potential bus users

**Comment by maxtkc** [comment]
This is a bigger issue than just that. Currently, we have
>Outbound-Inbound

This means I have to figure out where the buses go. Once Ezra sends the spreadsheet, I'll use that to hard code it in for now.

**Comment by jlawrenceCambridge** [comment]
Correct. Ezra should have sent that over this morning.  We need to use inbound on one screen and outbound on the other.

**Comment by maxtkc** [comment]
All of the definite stops should only post the destination, and not both. I'm going to figure out a better system for this soon, but first I wanted to hard-code it so we can get everything launched ASAP.

**Comment by livebusglocal** [comment]
The ones for 1369 Inman (1425 and 1404) and Dickson Bros (2076 and 20761) still have this problem.

![screenshot 2018-08-16 at 2 30 44 pm](https://user-images.githubusercontent.com/42448750/44228349-1a4eb480-a163-11e8-8070-a022f4c6be88.png)
![screenshot 2018-08-16 at 2 29 51 pm](https://user-images.githubusercontent.com/42448750/44228350-1a4eb480-a163-11e8-9011-a712e9acdeb7.png)
![screenshot 2018-08-16 at 2 28 36 pm](https://user-images.githubusercontent.com/42448750/44228351-1a4eb480-a163-11e8-8a94-ad56087d415f.png)

**Comment by maxtkc** [comment]
Sorry, yes. I put the Harvard stop aside and meant to get back to it later. I think it's best we wait on the Harvard bus tunnels and make sure to figure out the others first, as Ezra mentioned in another issue. I'll work on 1369 Inman (1425 and 1404) now.

**Comment by maxtkc** [comment]
Will be fixed in next big update (branch **auto_dest**). No more ~~hardcoding~~!
